cocos/platform/ios/CCDirectorCaller-ios.mm

```diff
@@ -75,6 +75,7 @@ - (instancetype)init
     if (self)
     {
         isAppActive = [UIApplication sharedApplication].applicationState == UIApplicationStateActive;
+        lastDisplayTime = 0;
```

Comment:
may be should change the code to invoke `init` function like this:
`s_sharedDirectorCaller = [[CCDirectorCaller new] init];` ?